SPECS/orangefs/orangefs.spec

```diff
@@ -89,7 +91,7 @@ export LDFLAGS="%{optflags} -Wl,--as-needed"
     --enable-fuse --disable-usrint --with-db-backend=lmdb
 %else
 %configure --enable-external-lmdb --enable-shared --disable-static \
-   --enable-fuse --disable-usrint --with-db-backend=lmdb --with-openib=/usr
+   --enable-fuse --disable-usrint --with-db-backend=lmdb --with-openib=%{_prefix}
```

Comment:
Let's remove the part for `armv7hl`. And could you use only one `%configure` argument per line?

Reply:
removed armv7hl and used only one line per configure.